pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -505,11 +507,15 @@ func SecretToLibvirtSecret(vm *v1.VirtualMachine, secretName string) string {
 	return fmt.Sprintf("%s-%s-%s---", secretName, vm.Namespace, vm.Name)
 }
 
-func QuantityToMegaByte(quantity resource.Quantity) Memory {
-	return Memory{
-		Value: uint(quantity.ToDec().ScaledValue(6)),
-		Unit:  "MB",
+func QuantityToByte(quantity resource.Quantity) (Memory, error) {
+	memorySize, _ := quantity.AsInt64()
+	if memorySize < 0 {
```

Comment:
you already added this check under validation, so you can remove

Reply:
I asked for that check. My reasoning was that we currently have nothing which checks if the validation webhook really works. I am unsure.

Reply:
Srry, missed your comment